Feat/#14: Connect with Android

penguin147 opened the pull request:
## Changes:running:

- Settings for connection with Android.

<br>

## Description:page_with_curl:

- To work on Android, we changed the way we load multiple paths and files.
- The .obj file outside of Asset Store was retrieved using "Obj Importer" in the AssetStore".
- Texture files were split into Byte, loaded, and merged again.
- ScriptableObject was created to set the path in Unity Engine without entering each Script.
- Crucially, it set up permissions to read external files in the Android environment.

<br>

## Note:exclamation:

1. Please check if there is a collision when driving on Android. @CHOIZION

<br>

penguin147 committed:
Chore: Import objImporter

Import objImport to import .obj from outside of asset.

Related to: #14, #15

penguin147 committed:
Feat: .obj Importer

The .obj file obtained from an external path using objImporter is used by unity as a game object.

Resolves: #15
Related to: #14

penguin147 committed:
Feat: Texture import

Split the png file into bytes and import it, then use LoadImage() to make it Texture2D.

Resolves: #16
Related to: #14

penguin147 committed:
Feat: Route Cleanup

Created FilePathManager to easily set up the path.

Resolves: #17
Related to: #14

penguin147 committed:
Feat: Import .obj files from Internal Storage

Import .obj file and texture files from internal storage. If you write down the internal storage path that contains the .obj file and the texture file in Settings/FilePath Manager, the file is retrieved from that location and rendered.

Resolves: #18
Related to: #14

CHOIZION commented:
테스트해보고 코멘트 남기겠습니다.
수고하셨습니다!

CHOIZION commented:
비디오 (READ_MEDIA_VIDEO)
오디오 (READ_MEDIA_AUDIO)
이미지 (READ_MEDIA_IMAGES)
내 모든 파일 권한 (MANAGE_EXTERNAL_STORAGE)
권한 추가하니 파일들이 잘 인식됩니다.
수고하셨습니다!

penguin147 committed:
Fix: Path unification

FilePathManager unified the path because the path of the object file and the texture file are the same.

Related to: #14
